core/src/main/java/org/elasticsearch/index/mapper/MapperBuilders.java

```diff
@@ -92,8 +95,11 @@ private MapperBuilders() {}
         return new DoubleFieldMapper.Builder(name);
     }
 
-    public static GeoPointFieldMapper.Builder geoPointField(String name) {
-        return new GeoPointFieldMapper.Builder(name);
+    public static BaseGeoPointFieldMapper.Builder geoPointField(String name, Version version) {
```

Comment:
Where are these mapper helpers used? I've only found them used by tests. Maybe we don't need them? I removed a number of them before for metadata fields.